Solutions/BitSight/Playbooks/GetWFHData/azuredeploy.json

```diff
@@ -0,0 +1,854 @@
+                                            "offset":  "@{variables('offset')}"
+                                        },
+                                        "uri":  "https://api.bitsighttech.com/ratings/v1/findings/wfh/"
+                                    }
+                                }
+                            },
+                            "runAfter":  {
+                                "Get_IP_address_from_incident":  [
+                                    "Succeeded"
+                                ]
+                            },
+                            "expression":  "@equals(variables('next_exist'), false)",
+                            "limit":  {
+                                "count":  60,
+                                "timeout":  "PT1H"
+                            },
+                            "type":  "Until"
+                        }
+                    },
+                    "outputs":  {
+                    }
+                },
+                "parameters":  {
+                    "$connections":  {
+                        "value":  {
+                            "azureloganalyticsdatacollector":  {
+                                "connectionId":  "[resourceId('Microsoft.Web/connections', variables('AzureloganalyticsdatacollectorConnectionName'))]",
+                                "connectionName":  "[variables('AzureloganalyticsdatacollectorConnectionName')]",
+                                "id":  "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/Azureloganalyticsdatacollector')]"
+                            },
+                            "azuresentinel":  {
+                                "connectionId":  "[resourceId('Microsoft.Web/connections', variables('MicrosoftSentinelConnectionName'))]",
+                                "connectionName":  "[variables('MicrosoftSentinelConnectionName')]",
+                                "id":  "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/Azuresentinel')]",
+                                "connectionProperties":  {
+                                    "authentication":  {
+                                        "type":  "ManagedServiceIdentity"
+                                    }
+                                }
+                            },
+                            "keyvault":  {
+                                "connectionId":  "[resourceId('Microsoft.Web/connections', variables('KeyvaultConnectionName'))]",
+                                "connectionName":  "[variables('KeyvaultConnectionName')]",
+                                "id":  "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/Keyvault')]"
+                            }
```

Comment:
Key vault connection should be managed service identity.

Kindly refer to Below playbook for ref 
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-BlockIP-URL/azuredeploy.json

Reply:
@manishkumar1991 
We have added managed service identity for Key vault for playbook.

Question:
Just to know can you please let me know what impact or extra steps is required for end user who is going to use/deploy this playbook once it publicly available?